src/scripts/main.js

```
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', function () {
+  const promise1 = new Promise((resolve, reject) => {
+    document.querySelector('.logo').addEventListener('click', () => {
+      resolve();
+    });
+  });
+
+  const promise2 = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(new Error('Promise was rejected after 3 seconds'));
+    }, 3000);
+  });
+
+  promise1
+    .then(() => {
+      const successMessage = document.createElement('div');
+
+      successMessage.className = 'message';
+      successMessage.textContent = 'Promise was resolved!';
+
+      document.body.appendChild(successMessage);
+    })
+    .catch(() => {});
```

The catch block for promise1 is empty. Consider handling the rejection case to avoid silent failures.